api-tests/operator_route_test.go

```diff
@@ -31,7 +31,7 @@ func TestOperatorRouteShouldReturnRigthOperatorVersion(t *testing.T) {
 		{"psmdb-operator", "1.10.0"},
 		{"psmdb-operator", "1.11.0"},
 		{"psmdb-operator", "1.12.0"},
-		{"pg-operator", "1.1.0"},
+		{"pg-operator", "1.3.0"},
```

Comment:
Shouldn't we have here `1.1.0`, `1.2.0` and `1.3.0` like we have for other operators?

Reply:
I guess we should